pkg/proxy/util/utils.go

```diff
@@ -477,6 +477,14 @@ func WriteLine(buf *bytes.Buffer, words ...string) {
 	}
 }
 
+// WriteRuleLine joins all words with spaces, terminate with newline and write to buff.
+func WriteRuleLine(buf *bytes.Buffer, chainName string, words ...string) {
+	if len(chainName) <= 0 || len(words) <= 0 {
+		return
```

Comment:
Does this actually come up anywhere? Oh, I see, `TestWriteLine` tests this behavior for `WriteLine` so you did the same for `WriteRuleLine`...

I'm not sure it's useful to have this case for `WriteRuleLine`... but I'm not sure it's _not_ useful... Keep it I guess, but
1. drop the `len(chainName)` part because calling `WriteRuleLine` with no chain name is just a bug
2. say `len(words) == 0` not `len(words) <= 0` because the length of an array can't be negative and probably some linter will eventually complain if you use `<=` here.

Reply:
yea this check was more of a "just in case".